node-admin/src/main/java/com/yahoo/vespa/hosted/node/admin/component/IdempotentTask.java

```diff
@@ -38,4 +38,22 @@
      * @throws RuntimeException (or a subclass) if the task is unable to converge.
      */
     boolean converge(T context);
+
+    /**
+     * Converge the task towards some some state where it can be suspended. The
+     * TaskContext should provide enough to determine what kind of suspend is wanted, e.g.
+     * suspension of only the task, or the task and the resources/processes it manages.
+     *
+     * suspendConverge() must be idempotent: it may be called any number of times, or
+     * interrupted at any time e.g. by `kill -9`.
+     *
+     * suspendConverge() is not thread safe: The caller must ensure there is at most one
```

Comment:
Do you mind adding a prefix &lt;p&gt; to each paragraph here and elsewhere in this file, as otherwise the javadoc formatting doesn't work as seen.